Formula/licenseplist.rb

```diff
@@ -0,0 +1,22 @@
+class Licenseplist < Formula
+  desc "License list generator of all your dependencies for iOS applications"
+  homepage "https://www.slideshare.net/mono0926/licenseplist-a-license-list-generator-of-all-your-dependencies-for-ios-applications"
+  url "https://github.com/mono0926/LicensePlist/archive/refs/tags/3.22.5.tar.gz"
+  sha256 "568c3bb40784ee59d5ebf12a6249d95f2eda2f2aabd43ad5be57e1c5db5f7b08"
+  license "MIT"
+
+  depends_on xcode: ["13.0", :build]
+  depends_on :macos
```

Comment:
Does it actually need macOS or does it just not build with the `swift` that we ship?

Reply:
The current implementation of LicensePlist depends on a library named APIKit which is not compatible with Linux's Swift 5.5 due to the `FoundationNetworking` issue.

I don't think the function of LicensePlist is specific to macOS and this restriction can be eventually removed.